conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -47,6 +47,11 @@ GET         /manage-transit-movements/test-only/view-arrival-notifications
 + nocsrf
 GET         /manage-transit-movements/test-only/view-departure-declarations                 controllers.testOnly.ViewAllDeparturesP5Controller.onPageLoad()
 
+
++ nocsfr
+GET         /manage-transit-movements/test-only/goods-under-control/:departureId   controllers.testOnly.TestOnlyGoodsUnderControlP5Controller.onPageLoad(departureId: String)
```

Comment:
Just released but I dont think we need the 'nocsfr' bit for pages. This is only needed for calls to the API. Might be worth trying to remove this.